### PR TITLE
SPCTR-1907 New Block - Modal

### PR DESCRIPTION
**Pull request by sushmak02**
### Description
The modal block is for simple things. For example, on the SureCart website which is built with Spectra, if I have an image with a play button on it, there is no way to have a video popup. We have to add a plugin for this seemingly standard thing. 

### Types of changes

New

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by AbhijitNage123 — APPROVED** [review]

File: .eslintrc.js [hunk 1]
```diff
@@ -39,6 +39,8 @@ module.exports = {
 		Cookies: true,
 		UAGBTableOfContents: true,
 		localStorage: true,
-		uagb_timeline_data: true
+		uagb_timeline_data: true,
+		UAGBModal: true,
+		UAGBCounter: true,
```
Comment (bsf-zanev) [inline .eslintrc.js:44]: Should these two entries in eslintrc.js exist? ( UAGBModal, UAGBCounter )

**Review by bsf-zanev — CHANGES_REQUESTED** [review]
Please address the comments below.

File: .eslintrc.js (hunk 1 above)
Comment (AbhijitNage123) [reply .eslintrc.js]: Removing UAGBCounter and keeping UAGBModal to resolve undefined UAGBModal while running the lint-js cmd.

**Review by bsf-zanev — APPROVED** [review]
Looks good to me.

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**